tuxemon/core/player.py

```diff
@@ -47,5 +47,7 @@ def __init__(self, npc_slug):
         super(Player, self).__init__(npc_slug)
         self.isplayer = True
 
-        # Game variables for use with events
-        self.game_variables = {}

```

Comment:
Maybe you shouldn't have removed this line?
        self.game_variables = {}

Reply:
Definitely shouldn't have removed that line.

Reply:
Thank you for pointing this out. Absent the other things I need to fix in this PR, can someone please clarify why this line shouldn't be removed? game_variables is already an attribute of tuxemon.core.npc.NPC, so if someone decided to modify the init value in there, wouldn't that get lost by overwriting in the child class `Player` here?

Reply:
Hmm... you make a good point, perhaps @bitcraft can weigh in on this?

Reply:
`game_variables` is defined in the NPC class, so its fine to remove.